modules/site_studio_config_management/config/install/cohesion.sync.settings.yml

```diff
@@ -0,0 +1,23 @@
+enabled_entity_types:
+  cohesion_base_styles: cohesion_base_styles
+  cohesion_color: cohesion_color
+  cohesion_component: cohesion_component
+  cohesion_component_category: cohesion_component_category
+  cohesion_content_templates: cohesion_content_templates
+  cohesion_custom_style: cohesion_custom_style
+  cohesion_font_library: cohesion_font_library
+  cohesion_font_stack: cohesion_font_stack
+  cohesion_helper: cohesion_helper
+  cohesion_helper_category: cohesion_helper_category
+  cohesion_icon_library: cohesion_icon_library
+  cohesion_master_templates: cohesion_master_templates
+  cohesion_menu_templates: cohesion_menu_templates
+  cohesion_scss_variable: cohesion_scss_variable
+  cohesion_style_guide: cohesion_style_guide
+  cohesion_style_guide_manager: cohesion_style_guide_manager
```

Comment:
cohesion_style_guide and cohesion_style_guide_manager should be added programatically based on module installed or not.

Reply:
Yes, this is left. Will work on and update PR.
> cohesion_style_guide and cohesion_style_guide_manager should be added programatically based on module installed or not.

Reply:
Done & updated PR.

Reply:
I just checked. this can be left as it is right now. This won't throw any error/warnings. Because the site_studio (before exporting the configurations) validates if current config entity is allowed to be exported. If yes, then export else skip. So, there's no harm in adding all allowable configuration here.

> cohesion_style_guide and cohesion_style_guide_manager should be added programatically based on module installed or not.